lab-lindsay/test/test.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const assert = require('assert');
+const greet = require('../lib/greet.js');
+
+describe('greet.js', function() {
+  describe('#greetName', function() {
+    it('should return hello lindsay if lindsay is assigned to name', function() {
+      assert.equal(greet.greetName('lindsay'), 'hello lindsay');
+    });
+    it('should return a valid string', function() {
+      assert.equal(typeof greet.greetName('lindsay'), 'string');
```

Comment:
Good practice checking data type. Think about if a number type value into a name field through some interface.  TDD allows us to check that and code defensively to alleviate any instances of user error that may break our functionality. What else can you think of to code defensively against?